components/ProfilePage/FollowingTab.tsx

```diff
@@ -133,3 +201,131 @@ export function FollowingTab({ className }: Props) {
     </>
   )
 }
+
+function FollowedBill({
```

Comment:
Can we consolidate `FollowedBill` and `FollowedOrg` into one component?

Reply:
done